CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ if (NOT DEFINED LXQT_DATA_DIR)
     )
 endif()
 
-set(LXQT_LIBRARY_NAME "lxqt" CACHE STRING "lxqt")
+set(LXQT_LIBRARY_NAME "lxqt2" CACHE STRING "lxqt2")
```

Comment:
@luis-pereira 
I think this instance of `lxqt` could be kept as it was (no change to `lxqt2`). The components that can be installed in parallel with Qt5-based ones shouldn't be affected, and the other components shouldn't be installed in parallel. Therefore, for example, there's no reason to have different `LXQT_TRANSLATIONS_DIR` or `LXQT_GRAPHICS_DIR` folders.

Am I missing something?

Reply:
I made some tests with globalshortcuts-wip_qt6:
 * translations will only be built and installed if  for all `*.ts` and `*.yaml` files,  `lxqt-` prefix is renamed  to `lxqt2-`.
 * If `LXQT_LIBRARY_NAME "lxqt"` is used translations are not loaded, they are looked for in `/usr/share/lxqt2/translations` by the applications.

So not sure what the best approach is -, renaming all files will break all weblate translations settings. Just not using any `lxqt2-` prefixes if port is finished?

Reply:
We should try remove `lxqt2` folder prefix and then try to build same library with both Qt6 and Qt5 and see if some files get overwritten or other kind of conflicts happen. This way you are sure there are no side effects.